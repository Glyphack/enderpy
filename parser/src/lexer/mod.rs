mod diagnostics;
pub mod fstring;
pub mod lexer;
