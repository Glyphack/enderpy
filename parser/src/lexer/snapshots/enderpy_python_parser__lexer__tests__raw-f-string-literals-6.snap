---
source: parser/src/lexer/mod.rs
description: "rf'''hello'''"
---
[
    Token {
        kind: RawFStringStart,
        value: Str(
            "rf'''",
        ),
        start: 0,
        end: 5,
    },
    Token {
        kind: FStringMiddle,
        value: Str(
            "hello",
        ),
        start: 5,
        end: 10,
    },
    Token {
        kind: FStringEnd,
        value: Str(
            "'''",
        ),
        start: 10,
        end: 13,
    },
]
